index.bs

```diff
@@ -3644,12 +3668,15 @@ The serialization algorithm works by appending successive byte strings to an, in
 1. Append [=CCDToString=]({{CollectedClientData/challenge}}) to |result|.
 1. Append 0x2c226f726967696e223a (`,"origin":`) to |result|.
 1. Append [=CCDToString=]({{CollectedClientData/origin}}) to |result|.
+1. If {{CollectedClientData/topOrigin}} is present:
+    1. Append 0x2c22746f704f726967696e223a (`,"topOrigin":`) to |result|.
+    1. Append [=CCDToString=]({{CollectedClientData/topOrigin}}) to |result|.
```

Comment:
This has to go after `crossOrigin`, as inserting it before that would be to "change the order in which those fields are serialized" which is prohibited in the "Future works" section below. Right, @agl?

Reply:
Fixed the order; see other comment for ongoing bigger-picture questions.